pynautobot/core/api.py

```diff
@@ -21,6 +21,7 @@
 from requests.adapters import HTTPAdapter
 from urllib3 import Retry
 
+import pynautobot
```

Comment:
We have a fear of possibly causing circular imports in the future, so please change this import to happen inside the `__init__` method and change it to only import `__version__`:

```python
    def __init__(
        ...
    ):
        from pynautobot import __version__  # pylint: disable=import-outside-toplevel

        base_url = "{}/api".format(url if url[-1] != "/" else url[:-1])
        ...
        self.http_session.headers.update({"User-Agent": f"python-pynautobot/{__version__}"})
```

Reply:
Not a problem; update committed to both branches.